### PR TITLE
[Agent] Add JDK13, JDK14 to build matrix

### PR DESCRIPTION
**Pull request by kezhenxu94**
Motivation:

Guarantee that the agent supports JDK13 and JDK14

Modifications:

Add JDK13 and JDK14 to build matrix

Result:

Build with JDK13 and JDK14

### CODE REVIEW

**Comment by wu-sheng** [comment]
Does this make the OAP running on the JDK 13/14 too?

**Comment by kezhenxu94** [comment]
> Does this make the OAP running on the JDK 13/14 too?

Nope, the OAP is running in the docker built by the `skywalking/docker/oap/Dockerfile.oap`, which is only based on `openjdk:8u181-jdk-stretch`

**Comment by wu-sheng** [comment]
BTW, I think you need to update the agent document about this?

File: docs/en/setup/service-agent/java-agent/README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Setup java agent
-1. Agent is available for JDK 8 - 12 in 7.x releases. JDK 1.6 - JDK 12 are supported in all 6.x releases [NOTICE¹](#notice)
+1. Agent is available for JDK 8 - 14 in 7.x releases. JDK 1.6 - JDK 14 are supported in all 6.x releases [NOTICE¹](#notice)
```
Comment (wu-sheng) [inline docs/en/setup/service-agent/java-agent/README.md]: We don't test 6.x for JDK 13/14. We should not indicate those.

**Pull request merged**